VogtPayroll5/PayrollConsoleReader.cs

```diff
@@ -24,5 +24,31 @@ public Employee GetEmployeeInfo()
 
             return emp;
         }
+
+        public bool ReturnTrueOrFalse()
```

Comment:
`Return` is very ambiguous term, given that it used as the say to return values in a function.

Maybe `ReadTrueOrFalse` would be more descriptive? Or even more explicitly -- `ReadTrueOrFalseFromConsole`